rust/feature-flags/src/flag_definitions.rs

```diff
@@ -94,8 +91,20 @@ pub struct FeatureFlag {
     pub ensure_experience_continuity: bool,
 }
 
+#[derive(Debug, Serialize, sqlx::FromRow)]
+pub struct FeatureFlagRow {
+    pub id: i32,
+    pub team_id: i32,
+    pub name: Option<String>,
+    pub key: String,
+    pub filters: serde_json::Value,
```

Comment:
I've opted to first parse filters out as just a json value, and then deserialise that into FlagFilters for better handling, and because it was getting pretty hard to try and coerce directly into the struct form I want.